ROADMAP.md

```diff
@@ -33,6 +33,12 @@ In the future, we want to allow third party tools to be plugged into Docker Swar
 of much bigger clusters.
 
 The work has started on defining the "cluster" interface in [#393](https://github.com/docker/swarm/issues/393)
-qThe first integration will be Mesos but other tools will arrive after, like kubernetes.
+
+The first integration will be Mesos but other tools will arrive after, like kubernetes.
 
 * [ ] Cluster Drivers
+
+Project Planning
+================
+* [0.2.0 Milestone](https://github.com/docker/swarm/wiki/0.2.0-Milestone-Project-Page)
```

Comment:
We should make it clear that each milestone (for instance 0.2.0) maps to a platform release (Docker 1.6)